dev/scripts/src/contract/transferContract.ts

```diff
@@ -0,0 +1,88 @@
+import { DappPayee } from '@prosopo/captcha-contract'
+import { ProviderEnvironment } from '@prosopo/env'
+import { Tasks } from '@prosopo/provider'
+import { defaultConfig, loadEnv } from '@prosopo/cli'
+import { exit } from 'process'
+import { getPairAsync } from '@prosopo/contract'
+import mongoose from 'mongoose'
+import path from 'path'
+
+interface iDapp {
+    email: string
+    name: string
+    url: string
+    account: string
+    marketingPreferences: boolean
+    createdAt: number
+}
+
+const dappSchema: mongoose.Schema<iDapp> = new mongoose.Schema({
+    email: String,
+    name: String,
+    url: String,
+    account: String,
+    marketingPreferences: Boolean,
+    createdAt: Number,
+})
+
+const Dapp = mongoose.model('Dapp', dappSchema, 'emails')
+
+const getAllMongoDapps = async (atlasUri: string) => {
+    await mongoose
+        .connect(atlasUri)
+        .then(() => console.log('Connected to MongoDB Atlas'))
+        .catch((err) => console.error('Error connecting to MongoDB:', err))
+
+    return await Dapp.find({})
+}
+
+// Doesn't work
+// const getAllContractDapps = async () => {
+// const config = defaultConfig()
+// const network = config.networks[config.defaultNetwork]
+// const secret = config.account.secret
+// const pair = await getPairAsync(network, secret)
+// const env = new ProviderEnvironment(config, pair)
+// await env.isReady()
+// const tasks = new Tasks(env)
+
+// const dapps = await (tasks.contract.contract as any)['dappContracts']()
+
+// return dapps
+// }
+
+// Function to register all dapps in contract
+const registerDapps = async (addresses: string[]) => {
+    const config = defaultConfig()
+    const network = config.networks[config.defaultNetwork]
+    const secret = config.account.secret
+    const pair = await getPairAsync(network, secret)
+    const env = new ProviderEnvironment(config, pair)
+    await env.isReady()
+    const tasks = new Tasks(env)
+
+    addresses.forEach(async (address) => {
+        await tasks.contract.query.dappRegister(address, DappPayee.dapp)
```

Comment:
Please add a value of the `dappStakeThreshold` here so that the dapp is activated on registration. Otherwise, we need to do this as a follow up when the user clicks the verify link (maybe that's better?).